Index: target/uiFailedTests.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>file:src/test/resources/features/NHS/userLogin.feature:8\n
===================================================================
diff --git a/target/uiFailedTests.txt b/target/uiFailedTests.txt
--- a/target/uiFailedTests.txt	
+++ b/target/uiFailedTests.txt	
@@ -1,1 +1,0 @@
-file:src/test/resources/features/NHS/userLogin.feature:8
